helper_methods.rb

```diff
@@ -292,3 +292,25 @@ def add_revision_statement(uri, description)
   post = @client.post(uri, resource.to_json)
   puts post.body
 end
+
+def get_all_resource_uris_for_institution()
+  #run through all repositories
+  resources_endpoints = []
+  repos_all = (3..12).to_a
+  repos_all.each do |repo|
+    resources_endpoints << 'repositories/'+repo.to_s+'/resources'
+    end
+  @uris = []
+  resources_endpoints.each do |endpoint|
+    ids_by_endpoint = []
+    ids_by_endpoint << @client.get(endpoint, {
```

Comment:
Could this just be an assignment instead of a push?
```
ids_by_endpoint = @client.get(endpoint, {...
```

Reply:
I'm not sure. I need to append to this array each time I cycle through a new repo.